android-bindings/docker/Dockerfile

```diff
@@ -28,14 +28,13 @@ RUN mkdir "$ANDROID_HOME" .android \
     && curl -o sdk.zip $SDK_URL \
     && unzip sdk.zip \
     && rm sdk.zip \
+    && mv cmdline-tools tools \
     && mkdir "$ANDROID_HOME/licenses" || true \
-    && ls "$ANDROID_HOME" \
     && echo "24333f8a63b6825ea9c5514f83c2829b004d1fee" > "$ANDROID_HOME/licenses/android-sdk-license" \
-    && echo "y" | $ANDROID_HOME/cmdline-tools/tools/bin/sdkmanager --licenses
+    && yes | $ANDROID_HOME/cmdline-tools/tools/bin/sdkmanager --licenses
 
 # Install Android Build Tool and Libraries
-RUN $ANDROID_HOME/cmdline-tools/tools/bin/sdkmanager --update
-RUN $ANDROID_HOME/cmdline-tools/tools/bin/sdkmanager "build-tools;${ANDROID_BUILD_TOOLS_VERSION}" \
+RUN $ANDROID_HOME/cmdline-tools/tools/bin/sdkmanager --install "build-tools;${ANDROID_BUILD_TOOLS_VERSION}" \
     "platforms;android-${ANDROID_VERSION}" \
     "platform-tools" \
```

Comment:
I believe we can remove this as the platform tools were already being installed from another installation.  I think this just installs a second copy and was throwing some kind of warning/error because it created a second directory.  I feel like this may have caused an issue with the build but not 100% sure.

Reply:
Removed it